easybuild/easyblocks/p/petsc.py

```diff
@@ -224,20 +228,44 @@ def configure_step(self):
                         withdep = "--with-%s" % dep[1].lower()
                     self.cfg.update('configopts', '%s=1 %s-dir=%s' % (withdep, withdep, deproot))
 
+            # SCOTCH has to be treated separately since they add weird postfixes
+            # to library names from SCOTCH 7.0.1 or PETSc version 3.17.
+            scotch = get_software_root('SCOTCH')
+            scotch_ver = get_software_version('SCOTCH')
+            if (scotch and LooseVersion(scotch_ver) >= LooseVersion("7.0.1")):
```

Comment:
The comparison should be vs `7.0` here imho:
```suggestion
            if (scotch and LooseVersion(scotch_ver) >= LooseVersion('7.0')):
```

Reply:
Done! :)